protocols.md

```diff
@@ -1,3 +1,101 @@
 ### Protocols
 
-*(These are mostly out of scope, but we need to describe at least one protocol to show how the layer above DIDComm builds on DIDComm constructs. This section should be short.)*
\ No newline at end of file
+*(These are mostly out of scope, but we need to describe at least one protocol to show how the layer above DIDComm builds on DIDComm constructs. This section should be short.)*
+
+
+
+#### Protocol Identifier URI
+
+A Protocol Type URI identifies protocol versions unambiguously.  Additionally, Protocol Type URIs may be used by a developer to locate documentation about a protocol. 
+
+The URI must be composed as follows:
+```
+[doc-uri][delim][protocol-name]/[semver]
+```
+With ABNF:
```

Comment:
We could probably define this by saying the type must be a URL, rather than providing an ABNF here.

Reply:
These comments should be made in #44, as this is a dependant PR. Sorry for the confusion